packages/catalog-model/examples/spotify-api.yaml

```diff
@@ -7,8 +7,11 @@ metadata:
     - spotify
     - rest
   annotations:
+    # The annotation is deprecated, we use placeholders (see below) instead, remove it later.
     backstage.io/definition-at-location: 'url:https://raw.githubusercontent.com/APIs-guru/openapi-directory/master/APIs/spotify.com/v1/swagger.yaml'
 spec:
   type: openapi
   lifecycle: production
   owner: spotify@example.com
+  definition:
+    $text: https://raw.githubusercontent.com/APIs-guru/openapi-directory/master/APIs/spotify.com/v1/swagger.yaml
```

Comment:
Realized we should link to `https://github.com/APIs-guru/openapi-directory/blob/master/APIs/spotify.com/v1/swagger.yaml` instead. That way the reader will be able to make authenticated requests and avoid ratelimiting

Reply:
Yes and no 😆 I can change that. But my goal was to demonstrate that it can be any url, even from another "location type" than the entity itself (which comes from GitHub). But my example comes still from GitHub in the end 😆

Reply:
Ah hmm, maybe link out to GitLab or smth? :p

The thing I wanted to avoid was people copying this and using raw URLs, because using the `github.com` one is generally better.

Reply:
> The thing I wanted to avoid was people copying this and using raw URLs, because using the github.com one is generally better.

Yeah, that's a good reason. For me it's fine to switch to the correct GitHub URL. I know now that it would work with another location. That spec isn't available somewhere else 😆